model/daal4py/dffml_model_daal4py/daal4pylr.py

```diff
@@ -0,0 +1,101 @@
+import pathlib
+import statistics
+import importlib
+import hashlib
+import os
+
+from typing import AsyncIterator, Tuple, Any, Type, List
+
+from dffml import (
+    config,
+    field,
+    entrypoint,
+    SimpleModel,
+    ModelNotTrained,
+    Accuracy,
+    Feature,
+    Features,
+    Sources,
+    Record,
+)
+
+
+@config
+class DAAL4PyLRModelConfig:
+    predict: Feature = field("Label or the value to be predicted")
+    features: Features = field("Features to train on. For SLR only 1 allowed")
+    directory: pathlib.Path = field(
+        "Directory where state should be saved",
+        default=pathlib.Path("~", ".cache", "dffml", "daal4py"),
+    )
+
+
+@entrypoint("daal4pylr")
+class DAAL4PyLRModel(SimpleModel):
+    CONFIG: DAAL4PyLRModelConfig
+
+    def __init__(self, parent) -> None:
+        super().__init__(parent)
+        self.d4p = importlib.import_module("daal4py")
+        self.pd = importlib.import_module("pandas")
+        self.np = importlib.import_module("numpy")
+        self.features = self.applicable_features
+        self._features_hash = self._feature_predict_hash()
+        self.lm = self.d4p.linear_regression_training(interceptFlag=True, streaming = True)
+        self.lm_predictor = d4p.linear_regression_prediction(streaming = True)
+        self.path = self.filepath(self.parent.config.directory, "trained_model.sav")
+        self.lm_trained = None
+        load_model()
+
+    def load_model(self):
+        if self.path.is_file():
+            self.lm_trained = self.joblib.load(self.path)
+
+    def filepath(self, directory, file_name):
+        return os.path.join(directory, file_name)
+
+    async def train(self, sources: Sources) -> None:
+        async for record in sources.with_features(
+            self.features + [self.parent.config.predict.NAME]
+        ):
+            xdata = record.features(self.features)
+            ydata = record.features(self.parent.config.predict.NAME)
+            self.lm.compute(xdata, ydata)
+        self.lm_trained = lm.finalize()
+        self.joblib.dump(self.lm_trained, self.path)
+
+    async def accuracy(self, sources: Sources) -> Accuracy:
+        if self.lm_trained is None:
+            raise ModelNotTrained("Train model before assessing for accuracy.")
+        data = []
+        async for record in sources.with_features(
+            self.features + [self.parent.config.predict.NAME]
```

Comment:
See https://github.com/intel/dffml/commit/6876ddb5c74c83fba51e314718ec30ff28c9dbd6

```suggestion
            self.features + [self.parent.config.predict.name]
```